Cargo.toml

```
@@ -16,37 +16,37 @@ include = ["../../LICENSE-APACHE", "../../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/rerun-io/rerun"
 rust-version = "1.67"
-version = "0.2.0"
+version = "0.3.0-alpha.0"
 
 [workspace.dependencies]
 # When using alpha-release, always use exact version, e.g. `version = "=0.x.y-alpha.z"
 # This is because we treat alpha-releases as incompatible, but semver doesn't.
 # In particular: if we compile rerun 0.3.0-alpha.0 we only want it to use
 # re_log_types 0.3.0-alpha.0, NOT 0.3.0-alpha.4 even though it is newer and semver-compatible.
-re_analytics = { path = "crates/re_analytics", version = "0.2.0" }
-re_arrow_store = { path = "crates/re_arrow_store", version = "0.2.0" }
-re_build_build_info = { path = "crates/re_build_build_info", version = "0.2.0" }
-re_build_info = { path = "crates/re_build_info", version = "0.2.0" }
-re_build_web_viewer = { path = "crates/re_build_web_viewer", version = "0.2.0" }
-re_data_store = { path = "crates/re_data_store", version = "0.2.0" }
-re_error = { path = "crates/re_error", version = "0.2.0" }
-re_format = { path = "crates/re_format", version = "0.2.0" }
-re_log = { path = "crates/re_log", version = "0.2.0" }
-re_log_types = { path = "crates/re_log_types", version = "0.2.0" }
-re_memory = { path = "crates/re_memory", version = "0.2.0" }
-re_query = { path = "crates/re_query", version = "0.2.0" }
-re_renderer = { path = "crates/re_renderer", version = "0.2.0" }
-re_sdk = { path = "crates/re_sdk", version = "0.2.0" }
-re_sdk_comms = { path = "crates/re_sdk_comms", version = "0.2.0" }
-re_smart_channel = { path = "crates/re_smart_channel", version = "0.2.0" }
-re_string_interner = { path = "crates/re_string_interner", version = "0.2.0" }
-re_tensor_ops = { path = "crates/re_tensor_ops", version = "0.2.0" }
-re_tuid = { path = "crates/re_tuid", version = "0.2.0" }
-re_ui = { path = "crates/re_ui", version = "0.2.0" }
-re_viewer = { path = "crates/re_viewer", version = "0.2.0" }
-re_web_viewer_server = { path = "crates/re_web_viewer_server", version = "0.2.0" }
-re_ws_comms = { path = "crates/re_ws_comms", version = "0.2.0" }
-rerun = { path = "crates/rerun", version = "0.2.0" }
+re_analytics = { path = "crates/re_analytics", version = "0.3.0-alpha.0" }
```

```suggestion
re_analytics = { path = "crates/re_analytics", version = "x=0.3.0-alpha.0" }
```

here and below. See the comment right above this line for why :]